package.json

```diff
@@ -8,7 +8,6 @@
   },
   "scripts": {
     "release": "yarn build && changeset publish",
-    "benchmark": "yarn workspace astro run benchmark",
```

Comment:
Not really in the scope of this PR but this script isn't relevant to this project